Documentation/designs/prometheus-operator-cli.md

```diff
@@ -0,0 +1,61 @@
+# Prometheus Operator CLI
+
+## Summary
+
+This document proposes a new CLI tool to manage Prometheus Operator resources, allowing users to create, update, and delete Prometheus Operator resources using a simple CLI interface as well as
+troubleshoot and debug Prometheus Operator resources.
+
+## Background
+
+During KubeCon Paris 2024, we received feedback from users that managing Prometheus Operator resources is difficult and error-prone. Users reported that they often encounter issues when creating, updating, and deleting Prometheus Operator resources, and that troubleshooting and debugging Prometheus Operator resources is challenging, below you can find some of the issues that users reported:
```

Comment:
Not only at KubeCon, but it has been a very old discussion between several members of the team in the past.

I remember talking about a CLI approach with @metalmatze and @paulfantom at least 2 years ago(even before I joined the team) 😛